test/util/stream_utils_unit.js

```diff
@@ -551,7 +551,8 @@ describe('StreamUtils', () => {
         });
       });
 
-      await shaka.util.StreamUtils.filterManifest(fakeDrmEngine, manifest);
+      await shaka.util.StreamUtils.filterManifest(fakeDrmEngine, manifest,
```

Comment:
If you put a default value in the function, it is not necessary to change these tests.